tests/unit/s2n_connection_test.c

```diff
@@ -124,7 +124,7 @@ int main(int argc, char **argv)
      */
     {
         /* Carefully consider any increases to this number. */
-        const uint16_t max_connection_size = 4250;
+        const uint16_t max_connection_size = 4264;
```

Comment:
I'm still not a fan of the unnecessary memory increase. Small increases have big impacts for customers with a lot of connections, and this change shouldn't actually need any extra memory.